### PR TITLE
2024-02-07 release

### PR DESCRIPTION
**Pull request by zie1ony**

### CODE REVIEW

File: newsletters/2024-02-07-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* [Zero-Knowledge Proofs: Questionnaire Result Verification in Smart Contracts](https://hackernoon.com/zero-knowledge-proofs-questionnaire-result-verification-in-smart-contracts) - article by Escholar. (02.02.2024)
+* [Scalable Collaborative zk-SNARK: Fully Distributed Proof Generation and Malicious Security](https://eprint.iacr.org/2024/143.pdf) - academic text by Xuanming Liu, Zhelei Zhou, Yinghao Wang, Bingsheng Zhang and Xiaohu Yang. (02.02.2024) 
+* [SNARK Recursion, Folding, and IVC](https://medium.com/@lucafra92/snark-recursion-folding-and-ivc-135b98458a4d) - article by Luca Franceschini. (02.02.2024)
+* [Sumchek 201](https://github.com/ingonyama-zk/papers/blob/main/sumcheck_201_chapter_1.pdf) - article by Ingonyama. (04.02.2024)
+* [Zero-Knowledge Proofs of Training for Deep Neural Networks](https://eprint.iacr.org/2024/162.pdf) - academic text by Kasra Abbaszadeh, Christodoulos Pappas, Dimitrios Papadopoulos and Jonathan Katz. (05.02.2024)
```
Comment (zie1ony) [inline newsletters/2024-02-07-newsletter.md:14]: Move this to ZKML

File: newsletters/2024-02-07-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* [zkMatrix: Batched Short Proof for Committed Matrix Multiplication](https://eprint.iacr.org/2024/161.pdf) - academic text by Mingshu Cong, Tsz Hon Yuen and Siu Ming Yiu. (05.02.2024)
+* [Mina to Ethereum ZK bridge](https://blog.lambdaclass.com/mina-to-ethereum-bridge/) - article by LambdaClass. (05.02.2024)
```
Comment (zie1ony) [inline newsletters/2024-02-07-newsletter.md:16]: top pick

File: newsletters/2024-02-07-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+* [Zero-Knowledge Proofs: A Solution to Privacy Concerns in Thorswap and More](https://hackernoon.com/zero-knowledge-proofs-a-solution-to-privacy-concerns-in-thorswap-and-more) - article by Mistakili. (05.02.2024)
+* [Verifiable evaluations of machine learning models using zkSNARKs](https://arxiv.org/pdf/2402.02675.pdf) - academic text by Tobin South, Alexander Camuto, Shrey Jain, Shayla Nguyen, Robert Mahari, Christian Paquin, Jason Morton and Alex 'Sandy' Pentland. (05.02.2024)
```
Comment (zie1ony) [inline newsletters/2024-02-07-newsletter.md:18]: Move to ZKML

File: newsletters/2024-02-07-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* [With New Arbitrum Orbit Integration, EigenDA and AltLayer Bring Horizontal Scalability to the Ethereum Ecosystem](https://www.blog.eigenlayer.xyz/eigenda-altlayer-arbitrum-orbit/). (02.02.2024)
+* [zkLink x NuLink](https://twitter.com/zkLink_Official/status/1753335777231302909). (02.02.2024)
+* [dYdX Chain Available for iOS](https://dydx.exchange/blog/dydx-chain-for-ios). (02.02.2024)
+* [Mina Protocol - Testworld Mission 2.0: Track 3 Retrospective](https://minaprotocol.com/blog/testworld-mission-2-0-track-3-retrospective). (02.02.2024)
+* [Scroll - Mainnet Weekly Update](https://twitter.com/Scroll_ZKP/status/1753573077848600994). (03.02.2024)
+* [Balancing Neutrality and Decentralization in EigenLayer](https://www.blog.eigenlayer.xyz/balancing-neutrality-and-decentralization-in-eigenlayer/). (05.02.2024)
+* [Axiom's component framework](https://twitter.com/timoethey/status/1754544968566485369). (05.02.2024)
+* [EZKL launches Lilith system](https://twitter.com/ezklxyz/status/1754610538095530368).
+* [Introducing Citrea: Bitcoin’s First ZK Rollup](https://www.blog.citrea.xyz/introducing-citrea/). (06.02.2024)
```
Comment (zie1ony) [inline newsletters/2024-02-07-newsletter.md:43]: top pick

File: newsletters/2024-02-07-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+### Podcasts and other talks
+* Weekly [Zero Knowledge Podcast](https://zeroknowledge.fm/309-2/) episode - *Episode 309: ZK Jargon Decoder with Nico Mohnblatt*. (31.01.2024)
+* [Strange Water Podcast](https://open.spotify.com/episode/7cp8soaEBBCQXY5AQb0fsG?si=c6949e189db942be) episode - *Building a ZK-Based Blockchain w/ Steve Pack (o1Labs)*. (01.02.2024) 
+* [Absolutely Zero Knowledge Podcast](https://www.youtube.com/watch?v=ML2yoy9wSxo/) episode - *Episode 5: The Convergance of ZKP and AI*. (02.02.2024) 
+* [Beacon Podcast (Web3 Unlocked)](https://www.youtube.com/watch?v=ZcG_BXZ8jHI) episode - *Victor Ji | Co-Founder of Manta Network: Building Real World Use-Cases with Zero Knowledge*. (06.02.2024)
+
+### Upcoming events
+* [Journal of Cryptology Call for Papers: Topical Collection on Modern Zero-Knowledge Protocols](https://iacr.org/jofc/TopicalCollection-mzkp.html) -  submissions will be open to March 1st, 2024. 
+* [ETHTaipei 2024](https://ethtaipei.org/) - Ethereum event on February/March 2024 in Taipei City, Taiwan.
+* [ETHSeoul 2024](https://www.ethseoul.org/) - conference on March 2024 in Seoul, South Korea.
```
Comment (zie1ony) [inline newsletters/2024-02-07-newsletter.md:60]: duplicate

**Review by zie1ony — COMMENTED** [review]
👍🏻

**Review by aleksanderziel — COMMENTED** [review]
Good

**Review by aleksanderziel — APPROVED** [review]
Good

**Pull request merged**